### ISSUE TITLE
Point source catalog in chains file

### ISSUE THREAD

**Issue by dncnwtts** [Cosmoglobe/Commander#96]
Would be nice to have the chains file containing an array of point source locations in the parameter output; partially to make it easier for the sky model code.

Should be simple, so I'm assigning myself.

**Comment by MetinSa** [comment]
Good idea. We could store this in the `parameters/radio` group of the `.h5` file. This group is created here: https://github.com/Cosmoglobe/Commander/blob/1d5c18f6cb50e0758a096c97ab8a318a28e751df/commander3/src/comm_output_mod.f90#L416
